### ISSUE TITLE
Merge in AbstractDoctrineMapper deprecated with ORM 3.0

### ISSUE THREAD

**Issue by CreativeNative** [Saeven/zf3-circlical-user#97]
Only for a reminder when upgrading to Doctrine ORM 3.0. Maybe we can fix this also earlier. There is no replacement, so I think it will be a simple save/persist. When we delete the update function we have a BC and when we change merge to persist we have duplicate code. I would go with the BC.

src/CirclicalUser/Mapper/AbstractDoctrineMapper.php
```
 public function update(object $entity): void
    {
        $this->getEntityManager()->merge($entity);
        $this->getEntityManager()->flush();
    }
```

**Comment by Saeven** [comment]
Makes sense.  I'll probably be forced to just make 'update' do the same as 'save' in this case, while I clean up a few apps on my side of the fence. Hundreds of such calls strewn throughout.

**Comment by CreativeNative** [comment]
Yes, no problem. I would deprecate the save function. Like this we can change the code slowly.
